deployment/publish.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+# publish.sh -- Deploy new containers to AWS ECR and restart ECS services.
+
+# Required encrypted environment variables:
+# -----------------------------------------
+# AWS_ACCESS_KEY_ID
+# AWS_SECRET_ACCESS_KEY
+# AWS_DEFAULT_REGION
+# AWS_STORAGE_BUCKET_NAME
+# AWS_LOCATION
+# AWS_ECR_URL
+# AWS_ECS_CLUSTER
+# AWS_ECS_SERVICE
+# ----------------------------------------
+
+set -euo pipefail
+
+if [ -z "$TRAVIS_PULL_REQUEST" ] || [ "$TRAVIS_PULL_REQUEST" == "false" ];
+then
+    if [ "$TRAVIS_BRANCH" == "develop" ];
+    then
+        # Rebuild the Grout image with production credentials.
+        cp ./grout-server/grout_server/settings.s3.py \
+           ./grout-server/grout_server/settings_deployment.py
+        docker build -t philly-fliers_grout:latest ./grout-server
+
+        # Collect and push static files to S3. This step needs to happen after
+        # the Grout image has been rebuilt, since the container for deploying
+        # static files depends on the production Grout image.
+        docker build -t philly-fliers_grout:deploy ./deployment/grout
+        docker run --env-file grout-server/.env \
+                   -e DEBUG="False" \
+                   -e INSTALLED_APPS="storages" \
```

Comment:
Just curious: why are installed apps parameterized?

Reply:
The variable name might be confusing here -- `INSTALLED_APPS` is an environment variable in the `.env` file that lets you add extra apps to the installed apps setting in `settings.py`. It's useful in this case since the production image has additional apps that aren't required for the development image. Maybe `EXTRA_INSTALLED_APPS` would be clearer?